src/components/Calculator.js

```diff
@@ -1,38 +1,74 @@
 /* eslint-disable */
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.

Reply:
@rloterh yes I know that because defining more than one class violates the linters rules and as you know we asked to use classes instead of hooks in that project till now though I forced to turn linter off in that file